src/include/test/unit_scoring.cc

```diff
@@ -36,7 +36,11 @@
 #include "detail/linalg/matrix.h"
 #include "scoring.h"
 
-#ifdef TILEDB_VS_ENABLE_BLAS
+#ifdef __AVX2__
+#include <immintrin.h>
+#endif
+
+#if defined(TILEDB_VS_ENABLE_BLAS) && 0
```

Comment:
(Minor) Could you add a comment or todo here with more context on why we disable this test?

Reply:
Done.

Reply:
@lums658 I do not see any commits to this PR before you landed.